.eslintrc.json

```diff
@@ -133,7 +133,7 @@
     "no-bitwise": "off",
     "no-buffer-constructor": "error",
     "no-caller": "error",
-    "no-catch-shadow": "error",
```

Comment:
why was this necessary ? It's okay to disable it for now, but we should file a task to re-enable it by turning disabling this for offending existing line

Reply:
@makarandp0 This is an error regarding Internet Explorer 8 and earlier:
```
error  Value of 'error' may be overwritten in IE 8 and earlier
```
So I don't think we need this check.